spec/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner_spec.rb

```diff
@@ -0,0 +1,83 @@
+describe ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner do
+  subject do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    auth                 = AuthToken.new(:name     => "test",
+                                         :auth_key => "valid-token",
+                                         :userid   => "jdoe",
+                                         :password => "password")
+    ems                  = FactoryGirl.create(:ems_hawkular,
+                                              :hostname        => 'localhost',
+                                              :port            => 8080,
+                                              :authentications => [auth],
+                                              :zone            => zone)
+    described_class.new(:ems_id => ems.id)
+  end
+
+  before do
+    allow_any_instance_of(ManageIQ::Providers::Hawkular::MiddlewareManager)
+      .to receive_messages(:authentication_check => [true, ""])
+    allow_any_instance_of(MiqWorker::Runner).to receive(:worker_initialization)
+  end
+
+  context "whitelist" do
```

Comment:
the common way to name instance method specs is `context "#whitelist?"`
for class methods it would be `context ".whitelist?"`